README.md

```diff
@@ -1,3 +1,7 @@
 [![codecov](https://codecov.io/gh/Whist-Team/Whist-Core/branch/main/graph/badge.svg)](https://codecov.io/gh/Whist-Team/Whist-Core)
+![PyPI](https://img.shields.io/pypi/v/whist-core)
+![PyPI - Downloads](https://img.shields.io/pypi/dm/whist-core)
+![PyPI - Python Version](https://img.shields.io/pypi/l/whist-core)
```

Comment:
Thats the license?

Reply:
It uses, what is defined here https://github.com/Whist-Team/Whist-Core/blob/67436921b6bc879d16cfdf057c5c3e0d3d2768b3/setup.py#L19

Yes `v` is version `dm` downloads/month and `l` license

Reply:
Oh now I saw it ... copy pasta.